extensions/istio/tests/src/test/java/io/fabric8/istio/test/v1beta1/VirtualServiceTest.java

```diff
@@ -106,83 +104,84 @@ void testCreate() throws InterruptedException {
+        + "\"match\":["
+        + "{\"uri\":{\"prefix\":\"/wpcatalog\"}},"
+        + "{\"uri\":{\"prefix\":\"/consumercatalog\"}}],"
+        + "\"name\":\"reviews-v2-routes\","
+        + "\"rewrite\":{\"uri\":\"/newcatalog\"},"
+        + "\"route\":[{\"destination\":{\"host\":\"reviews.prod.svc.cluster.local\",\"subset\":\"v2\"}}]}]}}",
+        recordedRequest.getBody().readUtf8());
   }
 
   @Test
   @DisplayName("Should Delete a Virtual Service")
   void testDelete() throws InterruptedException {
     server.expect().delete().withPath("/apis/networking.istio.io/v1beta1/namespaces/ns3/virtualservices/service3")
-      .andReturn(HttpURLConnection.HTTP_OK, new VirtualServiceBuilder().build())
-      .once();
+        .andReturn(HttpURLConnection.HTTP_OK, new VirtualServiceBuilder().build())
+        .once();
     Boolean deleted = client.v1beta1().virtualServices().inNamespace("ns3").withName("service3").delete();
     assertTrue(deleted);
 
     RecordedRequest recordedRequest = server.takeRequest();
-    assertEquals("{\"apiVersion\":\"v1\",\"kind\":\"DeleteOptions\",\"propagationPolicy\":\"Background\"}", recordedRequest.getBody().readUtf8());
+    assertEquals("{\"apiVersion\":\"v1\",\"kind\":\"DeleteOptions\",\"propagationPolicy\":\"Background\"}",
+        recordedRequest.getBody().readUtf8());
   }
 
   @Test
   @DisplayName("Should delete with PropagationPolicy=Orphan")
   void testDeleteOrphan() throws InterruptedException {
     server.expect().delete().withPath("/apis/networking.istio.io/v1beta1/namespaces/ns3/virtualservices/service3")
-      .andReturn(HttpURLConnection.HTTP_OK, new VirtualServiceBuilder().build())
-      .once();
+        .andReturn(HttpURLConnection.HTTP_OK, new VirtualServiceBuilder().build())
+        .once();
     Boolean deleted = client.v1beta1().virtualServices().inNamespace("ns3").withName("service3")
-      .withPropagationPolicy(DeletionPropagation.ORPHAN).delete();
+        .withPropagationPolicy(DeletionPropagation.ORPHAN).delete();
     assertTrue(deleted);
 
     RecordedRequest recordedRequest = server.takeRequest();
     assertEquals("{\"apiVersion\":\"v1\",\"kind\":\"DeleteOptions\",\"propagationPolicy\":\"Orphan\"}",
-      recordedRequest.getBody().readUtf8());
+        recordedRequest.getBody().readUtf8());
   }
 
   /*
-    ---
-apiVersion: "networking.istio.io/v1beta1"
-kind: "VirtualService"
-metadata:
-  annotations: {}
-  finalizers: []
-  labels: {}
-  name: "details"
-  ownerReferences: []
-spec:
-  hosts:
-  - "details"
-  http:
-  - route:
-    - destination:
-        host: "details"
-        subset: "v1"
-     */
+   * ---
+   * apiVersion: "networking.istio.io/v1beta1"
+   * kind: "VirtualService"
+   * metadata:
+   * annotations: {}
+   * finalizers: []
+   * labels: {}
+   * name: "details"
+   * ownerReferences: []
+   * spec:
+   * hosts:
+   * - "details"
+   * http:
+   * - route:
+   * - destination:
+   * host: "details"
+   * subset: "v1"
+   */
```

Comment:
Indentation was broken here by spotless :sob: 
Need to wrap with `<pre>{@code` ... `}</pre>`

Reply:
The config has 

```<setting id="org.eclipse.jdt.core.formatter.comment.format_block_comments" value="true"/>```

On something like this, I'd opt for just removing from the file - we generally don't have comments with the raw yaml anywhere else.

Reply:
I understand that <pre> enclosed Javadoc should not be altered.

Anyway, just remove it then. One way or the other, we shouldn't have invalid YAML.

Reply:
I've updated this to use formatter:off / on